packages/core/components/Modal/CodeSnippet/CodeSnippet.module.css

```diff
@@ -17,20 +17,6 @@
     min-width: 300px;
 }
 
-.button-menu, .button-menu button {
-    background-color: var(--secondary-background-color);
-    color: var(--secondary-text-color);
-}
-
-.button-menu i, .button-menu li > div {
-    color: var(--secondary-text-color);
-}
-
-.button-menu :is(a, button):hover, .button-menu button:hover i {
```

Comment:
<img width="465" alt="image" src="https://github.com/AllenInstitute/aics-fms-file-explorer-app/assets/24441706/1c5ce523-d85f-42f4-8f8e-5f60c6600daf">
Seeing a blue hover state for the code snippet language menu, unsure if intentional/can't remember if it was there before

Reply:
Strange, will fix